datahub-web-react/src/app/entity/shared/tabs/ML/MlPrimaryKeyFeatureTableTab.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useBaseEntity } from '../../EntityContext';
+import { EntityType } from '../../../../../types.generated';
+import { EntityList } from '../Entity/components/EntityList';
+import { useEntityRegistry } from '../../../../useEntityRegistry';
+import { GetMlPrimaryKeyQuery } from '../../../../../graphql/mlPrimaryKey.generated';
+
+export const FeatureTableTab = () => {
```

Comment:
Do we need 2 FeatureTableTab components?

Reply:
Its a bit simpler this way so we can keep things typesafe